go/host/container/host_container.go

```diff
@@ -108,7 +108,7 @@ func NewHostContainerFromConfig(parsedConfig *config.HostInputConfig) *HostConta
 	cfg.ID = ethWallet.Address()
 
 	fmt.Println("Connecting to the enclave...")
-	enclaveClient := enclaverpc.NewClient(cfg, logger)
+	enclaveClient := enclaveclient.NewEnclaveRPCClient(cfg, logger)
```

Comment:
I personally preferred the previous naming. Feels quite redundant now, and doesn't give out more information.